mobility/Mobility.m

```diff
@@ -306,6 +333,7 @@ function plotTrajectory(obj)
 			obj.wallDistance = 1;
 			obj.movementSpeed = obj.Velocity; % [m/s]
 			obj.pedestrianHeight = Config.Ue.height;
+			obj.shipHeight = Config.Ue.height;
 			if strcmp(obj.Scenario, 'pedestrian')
```

Comment:
Usually we use the "switch" statement instead of a long series of if and elseif.

Reply:
Thanks for the comment, will update